src/Bundler/Processor/TsContentProcessor.php

```diff
@@ -35,8 +35,8 @@ public function transpile(string $cwd, ContentItem $item): void
     {
         $module_name = $item->module_name;
 
-        if (false !== ($i = strrpos($module_name, '.'))) {
-            $module_name = substr($module_name, 0, $i);
+        if (preg_match('/\.ts$/i', $module_name)) {
+            $module_name = preg_replace('/\.ts$/i', '', $module_name);
```

Comment:
can you not always replace it? because `preg_replace` does not do anything if it does not match anything.